src/main/java/org/opentripplanner/standalone/Router.java

```diff
@@ -151,7 +151,11 @@ public void startup(JsonNode config) {
                 }
             }
         }
-
+        
+        JsonNode clusterByParentStation = config.get("clusterByParentStation");
+        if (clusterByParentStation != null && clusterByParentStation.isBoolean()) 
+        	graph.clusterByParentStation = clusterByParentStation.asBoolean();
```

Comment:
... and if it's not a boolean, throw an exception or something.

But actually, it should be a string called something like "clusterMode", with values "parentStation" and "geographic" -- maybe later someone will add something that clusters by name or something.